onepay/controllers/front/transaction.php

```diff
@@ -67,13 +69,14 @@ public function initContent() {
             }
 
             try {
-                $options = new Options();
+
+                $options = new Options($apiKey, $sharedSecret);
```

Comment:
same question here.

Reply:
is to omolog all the plugins, they are all like this

Reply:
Changed